Assignment/solution_student/index.html

```diff
@@ -8,24 +8,88 @@
+            </ul>
+          </nav>
+        </div> 
       </header>
 
+      <div class="main-content">
       <main>
-        <p>Our HTML starter template!</p>
+        <section class="article-container">
+          <article class="article">
+            <a class="headline-link" href="#">How I implemented Relaxr in 2 weeks and changed my life</a>
+            <img src="images/blog_photo1.jpg" alt="people rowing boats">
+            <p><span class="drop-cap">L</span>orem ipsum dolor sit amet, interdum suspendisse lacus, sed dolor, sit eu a enim nec rhoncus, in lectus. Illo commodo condimentum eget est aliquam voluptatum, tempus nonummy phasellus tortor, interdum diam, pellentesque sit leo dictumst morbi, lacus est dignissim purus vitae conubia. Curabitur numquam dignissim sem su  scipit vestibulum. Elementum a nunc sit dui, hendrerit metus vel eu suscipit. Illo commodo condimentum eget est aliquam voluptatum, tempus nonummy phasellus tortor, interdum diam.</p>
+            <a href="#"><div class="read-more">Read More »</div></a>
+            <hr>
+          </article>
+
+          <article class="article">
+          <a class="headline-link" href="#">I traveled across the globe thanks to Relaxr’s Excel auomation</a>
+          <img src="images/blog_photo2.jpg" alt="empty boats on the water">
+          <p><span class="drop-cap">L</span>orem ipsum dolor sit amet, interdum suspendisse lacus, sed dolor, sit eu a enim nec rhoncus, in lectus. Illo commodo condimentum eget est aliquam voluptatum, tempus nonummy phasellus tortor, interdum diam, pellentesque sit leo dictumst morbi, lacus est dignissim purus vitae conubia. Curabitur numquam dignissim sem suscipit vestibulum. Elementum a nunc sit dui, hendrerit metus vel eu suscipit. Illo commodo condimentum eget est aliquam voluptatum, tempus nonummy phasellus tortor, interdum diam.</p>
+          <a href="#"><div class="read-more">Read More »</div></a>
+          <hr>
+          </article>
+        </section>  
       </main>
+      </div>
+
+      <aside>
+        <div class="categories">
+          <h2>Categories</h2>
+          <ul>
+            <li><a href="#">Success stories</a></li>
+            <li><a href="#">Stats</a></li>
+            <li><a href="#">How tos</a></li>
+            <li><a href="#">Best of</a></li>
+            <li><a href="#">Business tips</a></li>
+          </ul>
+        </div>  
+        <div class="about">
+        <h2>About</h2>
+          <p>Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.</p>
+          <a href="#"><div class="learn-more">Learn More »</div></a>
+        </div>
+        <div class="ad-unit">
+          <img src="images/ad-unit.png" alt="advertisement">
+        </div>
+      </aside>
+      
+      <section class="footer-wrapper">
+        <section class="sign-up">
+          <a href="#"><div class="button-secondary">Sign Up Now!</div></a>
```

Comment:
The `div` here seems unnecessary — it's a little strange to put a block-level element inside of an inline element for one, but I think it's unneeded anyways, you should be able to just give the `a` tag the `button-secondary` class itself and be okay —  you'll just have to give it `display: block;` so that you can apply the width and padding and keep it centered :)